src/stat.cc

```diff
@@ -11,6 +11,7 @@
 #include "squid.h"
 #include "CacheDigest.h"
 #include "CachePeer.h"
+#include "ClientActiveRequests.h"
```

Comment:
AFAICT, we should move statClientRequests() (and its cache manager registration) from `stat.cc` into the new ClientActiveRequests home instead. I think this move should be done in any case, but I only insist on this move if you insist on adding new source files for ClientActiveRequests -- if you want to treat ClientActiveRequests as a stand-alone "feature", let's not yank an important piece of it out.

Reply:
done